CHANGES.md

```diff
@@ -44,6 +44,7 @@
 * Change the method to make copies of the estimator in `ExponentiatedGradient` from `pickle.dump` to `sklearn.clone`. 
 * Add an argument to GridSearch and ExponentialGradient `sample_weight_name` to control
   how `sample_weight` is supplied to `estimator.fit`.
+* Large changes to the metrics API. Please see the documentation and examples for more information.
```

Comment:
if we are dropping functions like all of the automatically derived metrics and all of the `make_*`, we should say so!

Reply:
I can add the derived metrics we generated back in, using the old backend. Adding a 'how to transition' document is a follow up.

Reply:
ok. as long as we have the "transitioning section" i'm good.

Reply:
actually--i think it would still be good to mention: which functions we are dropping, and which functions are changing their API. so for now say: "MetricFrame is introduced, and group_summary variants are being removed."

Reply:
Done